catroid/src/main/res/values-ur/strings.xml

```diff
@@ -1771,7 +1771,7 @@
     <string name="formula_editor_logic_comparison">موازنہ آپریٹرز</string>
     <string name="formula_editor_object_general">عام خصوصیات</string>
     <string name="formula_editor_object_movement">حرکت خصوصیات</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations